src/index.js

```diff
@@ -5,17 +5,23 @@ import * as serviceWorker from "./serviceWorker";
 import "semantic-ui-css/semantic.min.css";
 import axios from "axios";
 import { BrowserRouter } from "react-router-dom";
-import { StripeProvider } from 'react-stripe-elements'
+import { StripeProvider } from "react-stripe-elements";
 import "./css/index.css";
+import { Provider } from "react-redux";
+import configureStore from "./state/store/configureStore";
 
 axios.defaults.baseURL = "http://localhost:3000/api";
+const store = configureStore();
+window.store = store;
```

Comment:
Should we have this line in our implementation?

Reply:
No!